docs/pages/server-access/troubleshooting-server.mdx

```diff
@@ -0,0 +1,145 @@
+This issue might be caused by systems that prevent newly installed software from being executable 
+by any user.  
+
+### Symptom
+
+This issue results in a disconnected session and an error message similar to the following:
+
+```text
+Failed to launch: fork/exec /proc/self/exe: permission denied.
+Process exited with status 255
+```
+
+### Solution
+
+You should check the permission settings for the `teleport` service. 
+
+To check the permissions on the teleport service:
+
+1. Open a terminal shell on the computer where you have installed the `teleport` service.
+
+1. Determine the location of the Teleport binary by running the following command:
+   
+   ```code
+   which teleport
+   ```
+
+1. Check the permission settings for the binary by running the following command:
+   
+   ```code
+   ls -al <Var name="/usr/local/bin/teleport" />
+   ```
+
+   The command should return output similar to the following:
+   
+   ```text
+   -rwxr-xr-x  1 root  wheel  531849504 Aug 30 18:32 /usr/local/bin/teleport
+   ```
+   
+   If you don't see the permission that allows other users to read and execute (-rwxr-x**r-x**), 
+   you should update the permissions. For example:
+
+   ```code
+   chmod o+rx teleport
+   ```
+   
+1. Restart the `teleport` service.
+
+## Missing logins for single sign-on users
+
+If you use an external identity provider to enable single sign-on for users, you should be sure
+to assign logins for those users.
+
+### Symptom
+
+Users who have access to Teleport through an authentication connector for an external identity
+provider don't see any of the logins they need to access remote resources.
+
+### Solution
+
+To fix this issue, you can assign logins to your single sign-on users or modify the traits in 
+the Teleport roles assigned to users through their group membership in the external identity provider.
+
+## Listing available servers
+
+In some cases,  running `tsh ls` or `tctl nodes ls` commands might include servers that are offline.
+For example, a server that has stopped sending a heartbeat to the Teleport Proxy Service
+might continue to be listed as available in the output of `tsh ls` or `tctl nodes ls` commands for
+10 minutes or more.
+
+### Symptom
+
+An unresponsive server is listed as available in the output of `tsh ls` or `tctl nodes ls` commands.
+For example, you might run either of these commands and see output similar to the following for
+servers that have previously sent a heartbeat signal to the Teleport Proxy Service even if
+one of these servers subsequently went offline:
+
+```code
+Node Name      Address        Labels                  
+-------------- -------------- ----------------------- 
+ip-172-3-1-242 127.0.0.1:3022 hostname=ip-172-3-1-242 
+ip-172-3-1-75  ⟵ Tunnel       hostname=ip-172-3-1-75  
+ip-172-3-2-177 ⟵ Tunnel       hostname=ip-172-3-2-177
+```
+
+### Solution
+
+To investigate whether an unresponsive server is sending a heartbeat, you can run the 
```

Comment:
```suggestion
To investigate whether a server that sent a heartbeat has become unresponsive, you can run the 
```
To be consistent with the rest of this section: the server sent a heartbeat, then became unresponsive.